app/server/api/search_api.py

```diff
@@ -31,33 +32,110 @@ def get(self):
         Return Value:
             Results object, similar to the existing transfer_accounts and credit_transfers API return values
         """
-        search_string = request.args.get('search_string') or ''
+        # HANDLE PARAM : search_stirng
+        search_string = request.args.get('search_string') or ''      
 
+        # HANDLE PARAM : search_type
         # Valid search types are: `transfer_accounts` and `credit_transfers`
+        # Default: transfer_accounts
         search_type = request.args.get('search_type') or 'transfer_accounts'
         if search_type not in ['transfer_accounts', 'credit_transfers']:
             response_object = {
                 'message': 'Invalid search_type \'{}\'. Please use type \'transfer_accounts\' or \'credit_transfers\''.format(search_type),
             }
             return make_response(jsonify(response_object)), 400
 
+        # HANDLE PARAM : sorting_options
+        # Valid params differ depending on sorting_options. See: sorting_options
+        # Default: rank
```

Comment:
handle param is `sort_by` not `sorting_options `, can you please doc string/comment ....